syntax = "proto3";
package metisfl;

import "google/protobuf/timestamp.proto";

message Ack {
  bool status = 1;
  google.protobuf.Timestamp timestamp = 2;
  string message = 3;
}

message GetHealthStatusResponse {
  map<string, bool> services_status = 1;
}

message ShutDownRequest {
  //TODO(stripeli): require some sort of auth key here?
}
